hera_cal/reflections.py

```diff
@@ -38,9 +38,30 @@
 
     V_12^data = V_12^model g_1^eff g_2^eff*
 
+----------------
+
 Reflections that couple other voltage signals into v_1 cannot
 be described in this formalism (for example, over-the-air aka
-inter-feed reflections).
+feed-to-feed reflections). We refer to these as "cross-reflections"
+and their behavior is to introduce a copy of the autocorrelation
+into the cross correlation. If an antenna voltage becomes:
+
+    v_1 <= v_1 + eps_1 v_2
+
+then the cross correlation is proportional to
+
+    V_12 = v_1 v_2^* + eps_1 v_2 v_2^* + ...
+
+where we see that the V_22 = v_2 v_2^* autocorrelation is now
+present in the V_12 cross correlation. V_22 is a non-fringing
+term but time-dependent term, meaning it will introduce an
+offset in the visibility at a specific delay that will have
+some time-dependent amplitude. This cannot be corrected
+via antenna-based calibration. However, we can in principle
+subtract this off at the visibility level by taking
+some form of a time-average and subtracting off the bias.
+The code here models the time and delay dependent behavior
+through a combination of SVD and fringe-rate filtering.
```

Comment:
Given the very large number of things you potentially attach to this object over the course of your various reflection fitting, I'd consider doing two things:

1) Add a list of all the things you attach to the object to this docstring, what they are, and explain which functions generate them.
2) Add an `__init__` that uses `super` but also sets all these things to `None`

Reply:
1. Sure thing

2. Hmm, I don't see how setting them to `None` by default helps clarity at all: I don't think its useful to have a bunch of `None` attributes for a user who, given their use-case, may not even create those structures anyways. Something like what `pyuvdata` does, which is to have them by default set to an object that is empty but holds a doc-string and other metadata would be helpful, but I think is massive overkill for the stage of this code right now.